velox/exec/RowContainer.h

```diff
@@ -187,6 +187,19 @@ class RowContainer {
             pool,
             ContainerRowSerde::instance()) {}
 
+  ~RowContainer();
+
+  /// makes 'this' share the string allocator of 'other'. This is
+  /// needed in unspilling, when aggregates are read from the original
+  /// container and added to a different RowContainer for merging. The
+  /// same aggregate functions are used for both reading the existing
+  /// unspilled data and for aggregating the merge. Because these are
+  /// the same aggregates, these need the same allocator for their
+  /// variable length data.
+  void shareStringsWith(RowContainer& other) {
```

Comment:
This API doesn't look safe as it can be problematic to call it after adding something to the container. It would be safer to pass the arena to the constructor.